### PR TITLE
Update message-handlers.ts

### PR DESCRIPTION
**Pull request by zzackey**
added uni

### CODE REVIEW

**Review by promise — CHANGES_REQUESTED** [review]
still wouldn't work

**Comment by zzackey** [comment]
I think I did it. Probably!

File: src/message-handlers.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export async function handleMessage(client: Client, message: Message) {
     return
   }
 
-  if ((message.content.toLowerCase().includes('yoni')) or (message.content.toLowerCase().includes('uni')) or (message.content.toLowerCase().includes('yone')) {
+  if ((message.content.toLowerCase().includes('yoni')) ||  (message.content.toLowerCase().includes('uni')) ||  (message.content.toLowerCase().includes('yone')) {
```
Comment (yonilerner) [inline src/message-handlers.ts:8]: - you dont need the extra parens
- youre calling `message.content.toLowerCase()` 3 times - could be saved in a variable
- this is too sensitive to things like `@everyone` - word break regexes should be used probably. once using regex, u dont need to lowercase it
- what if we want to add more words? I'd suggest an array of words you want to check for, and then check if any of them are in the list:

```ts
// outside of `handleMessage` function
const wordsToReactTo = [/\byoni/i, /\buni/i, /\byone/i]
...
// inside of `handleMessage` function
if (wordsToReactTo.some(regex => regex.test(message.content)) {
...
}
```

File: src/message-handlers.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
+
```
Comment (appellation) [inline src/message-handlers.ts:1]: This is critical. Huge QoL improvement.

**Review by appellation — APPROVED** [review]

**Pull request merged**